package-lock.json

```diff
@@ -1,114 +1,293 @@
 {
   "name": "contentful-sdk-jsdoc",
   "version": "2.2.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Hmmm do we want to introduce version 2?

Reply:
Okay, found this thread, will revert it: https://contentful.slack.com/archives/C06UV45E3/p1624287602135500